PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -1644,6 +1644,17 @@ bool FindReplaceDlg::processReplace(const TCHAR *txt2find, const TCHAR *txt2repl
 				msg += moreMatches?TEXT("The next occurence found"):TEXT("The next occurence not found");
 				setStatusbarMessage(msg, FSMessage);
 			}
+
+			if (pTextFind)
+			{
+				delete[] pTextFind;
+				pTextFind = nullptr;
+			}
+			if (pTextReplace)
+			{
+				delete[] pTextReplace;
+				pTextReplace = nullptr;
+			}

```

Comment:
What the point of pTextFind and pTextReplace local variables in the first place? It seems to me that they can be deleted completely - txt2find and txt2replace can be used directly instead. Am I wrong?